### PR TITLE
Update MAINTAINERS.md

### PR DESCRIPTION
**Pull request by hangrychae**
Names in correct alphabetical order

### CODE REVIEW

**Comment by gowriaddepalli** [comment]
Can you please add reviewers? Thanks

File: MAINTAINERS.md [hunk 1]
```diff
@@ -3,15 +3,15 @@
 Following is the current list of maintainers on this project
 
 The maintainers are listed in alphabetical order.
-- Michael Hind ([michaelhind](https://github.com/michaelhind))
-- Sam Hoffman ([hoffmansc](https://github.com/hoffmansc))
+- Animesh Singh ([animeshsingh](https://github.com/animeshsingh))
```
Comment (gowriaddepalli) [inline MAINTAINERS.md:6]: Nice catch!

Comment (hangrychae) [reply MAINTAINERS.md:6]: Thanks! Coming from GHC Open Source workshop :)

**Comment by mnagired** [comment]
Thanks @hangrychae! Unfortunately, looks like there is an identical PR that was approved here: #464.

If you don't mind, please go through that PR and, if you agree with it, leave a comment and I'll make sure you are also added to the list of assignees so that you also get credit for your work :)

**Comment by mnagired** [comment]
closing this PR due to duplicate PR: #464

**Comment by hangrychae** [comment]
> Thanks @hangrychae! Unfortunately, looks like there is an identical PR that was approved here: #464.
> 
> If you don't mind, please go through that PR and, if you agree with it, leave a comment and I'll make sure you are also added to the list of assignees so that you also get credit for your work :)

Thanks! Just checked the other pull request.

**Pull request closed without merging**